messaging/src/main/java/org/axonframework/eventhandling/EventTrackerStatus.java

```diff
@@ -74,4 +76,21 @@
      * @return the exception that caused processing to fail, or {@code null} when processing normally
      */
     Throwable getError();
+
+    /**
+     * Return the estimated relative position this tracking token represents for the event tracker status whether this Segment is still replaying.
+     * In case no estimation can be given or no replay an {@code OptionalLong.empty()} will be returned.
+     *
+     * @return the estimated relative position this tracking token represents for the event tracker status whether this Segment is still replaying
+     */
+    OptionalLong getReplayPosition();
```

Comment:
is it really useful? getTrackingToken().getPosition() and is Replaying() together can give the same information I think, is it correct?

Reply:
The idea behind the ticket is to promote and get a much more clear way to access already existing information.

Reply:
I agree, it seems that currentPosition and reset(Token)Position should be enough